packages/features/bookings/Booker/utils/event.ts

```diff
@@ -16,9 +16,10 @@ import { useBookerStore } from "../store";
  */
 export const useEvent = () => {
   const [username, eventSlug] = useBookerStore((state) => [state.username, state.eventSlug], shallow);
+  const isTeamEvent = useBookerStore((state) => state.isTeamEvent);
```

Comment:
the `useEvent` hook takes the `isTeamEvent` boolean from the store and automatically passes that to the query so it fetches the right data.